### PR TITLE
Continue batch task for the rest of the rows

### PR DESCRIPTION
**Pull request by alex-deaconu**
@ezequielc please review. Thanks!

### CODE REVIEW

File: src/main/java/com/risevision/storage/servertasks/LockDownUnpaidBucketsServerTask.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ void handleRequest() throws IOException {
           taskList.add(makeTask(companyId));
         }
       } catch (ServiceFailedException e) {
-        throw new IOException("SubscriptionStatusFetcher error");
+        // Continue with the rest of the rows
```
Comment (ezequielc) [inline src/main/java/com/risevision/storage/servertasks/LockDownUnpaidBucketsServerTask.java]: @alex-deaconu Would it be worth to add a log here with company id and error code? Cheers

**Comment by ezequielc** [comment]
@alex-deaconu Just one question. Looks good anyway. :+1: Thanks!

File: src/main/java/com/risevision/storage/servertasks/LockDownUnpaidBucketsServerTask.java (hunk 1 above)
Comment (alex-deaconu) [inline src/main/java/com/risevision/storage/servertasks/LockDownUnpaidBucketsServerTask.java]: @ezequielc good question. I thought about it however the `getStoreAuthorization` function call earlier has quite specific logging:
https://github.com/Rise-Vision/storage-server/blob/master/src/main/java/com/risevision/storage/api/StoreAuthorizationFetcherImpl.java#L49

Not sure if it's necessary. Thoughts?

Thanks.

Comment (ezequielc) [inline src/main/java/com/risevision/storage/servertasks/LockDownUnpaidBucketsServerTask.java]: @alex-deaconu I don't think it is necessary in this case. Thanks!

**Pull request merged**